spec/amp-story-player.md

```diff
@@ -0,0 +1,201 @@
+  <tr>
+    <td width="40%"><strong>Required Scripts</strong></td>
+    <td>
+    <code>&lt;script async src="https://cdn.ampproject.org/amp-story-player-v0.js">&lt;/script></code>
+    <code>&lt;link href="https://cdn.ampproject.org/amp-story-player-v0.css" rel='stylesheet' type='text/css'></code>
+    </td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><ul>
+      <li>See <a href="https://github.com/ampproject/amphtml/blob/master/examples/amp-story/player.html">code snippet</a>.</li>
+    </ul></td>
+  </tr>
+</table>
+
+[TOC]
+
+## AMP story player usage
+
+Use `amp-story-player` to embed and play stories within a webpage.
+
+### Embed in a non-AMP page
+
+The following example demonstrates a basic embed of <amp-story-player> in a non-AMP webpage.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script
+      async
+      src="https://cdn.ampproject.org/amp-story-player-v0.js"
+    ></script>
+    <link
+      href="https://cdn.ampproject.org/amp-story-player-v0.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+  </head>
+  <body>
+    <h1>This is a NON-AMP page that embeds a story below:</h1>
+    <amp-story-player style="width: 360px; height: 600px;">
+      <a
+        href="https://example.com/story.html"
+        style="--story-player-poster: url('https://example.com/img/promo3x4.jpg');"
+      >
+        <span>A description for my story.</span>
+      </a>
+    </amp-story-player>
+  </body>
+</html>
+```
+
+The high-level API for the player looks like the following:
+
+[example preview="top-frame" playground="true"]
+
+```html
+<head>
+  <script
+    async
+    src="https://cdn.ampproject.org/amp-story-player-v0.js"
+  ></script>
+  <link
+    href="https://cdn.ampproject.org/amp-story-player-v0.css"
+    rel="stylesheet"
+    type="text/css"
+  />
+  <style injected>
+    .story-player-iframe {
+      background-size: cover;
+      display: block;
+      position: relative;
+      height: 100%;
+      width: 100%;
+      flex: 0 0 100%;
+    }
+
+    .title {
+      background: linear-gradient(
+        to bottom,
+        rgba(0, 0, 0, 0),
+        rgba(0, 0, 0, 1)
+      );
+      display: block;
+      position: absolute;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      color: #fff;
+      font-size: 40px;
+      padding: 88px 32px 32px 32px;
+      line-height: 1.25;
+      text-shadow: 3px 3px #000;
+    }
+  </style>
+</head>
+<body>
+  <h1>This is a NON-AMP page that embeds a story below:</h1>
+  <amp-story-player style="width: 360px; height: 600px;">
+    <a
+      href="https://www-washingtonpost-com.cdn.ampproject.org/v/s/www.washingtonpost.com/graphics/2019/lifestyle/travel/amp-stories/a-locals-guide-to-what-to-eat-and-do-in-mexico-city/"
+      data-poster-portrait-src="https://www-washingtonpost-com.cdn.ampproject.org/v/s/www.washingtonpost.com/graphics/2019/lifestyle/travel/amp-stories/a-locals-guide-to-what-to-eat-and-do-in-mexico-city/img/promo3x4.jpg"
```

Comment:
I also see this has ` data-poster-portrait-src="https://www-washingtonpost....` instead of ` style="--story-player-poster: url('https://example.com/img/promo3x4.jpg');"`?

Reply:
Good catch!